lib/common/socket/evloop.c.h

```diff
@@ -19,11 +19,58 @@
  * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
  * IN THE SOFTWARE.
  */
+
+#include <stdlib.h>
+#ifdef _WIN32
+
+struct iovec {
+   char *buf;
+   unsigned long len;
+};
+
+static ssize_t
+writev(SOCKET fd, const struct iovec *iov, int iovcnt) {
```

Comment:
Thank you for this I didn't spot this.

done: haven't pushed changes yet as we need to address all the issues